src/main/java/appeng/util/item/AEFluidStack.java

```diff
@@ -372,22 +367,24 @@ public Fluid getFluid()
 	}
 
 	@Override
-	void writeIdentity( final ByteBuf i ) throws IOException
+	public ItemStack getDisplayStack()
 	{
-		final byte[] name = this.fluid.getName().getBytes( "UTF-8" );
-		i.writeByte( (byte) name.length );
-		i.writeBytes( name );
+		// TODO: fluids, how do they even work?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)